README.md

```diff
@@ -67,11 +70,14 @@ submissions that are stored in Synapse.
 Running the application locally
 -------------------------------
 
+Set localDevelopment variable to "TRUE" in "app.R" file of main directory.
+
 ``` r
-library("stopadforms")
-run_app()
+localDevelopment = TRUE
 ```
 
```

Comment:
Add info here about the config file you need for local development too! You can either just document it here, or also check in a 'clean' version of it (no secrets) that people can populate locally.

Reply:
added clean config.yml file